zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepo.java

```diff
@@ -67,7 +67,7 @@
   /**
    * Release any underlying resources
    */
-  @ZeppelinApi public void close();
+  @ZeppelinApi public abstract void close();
```

Comment:
abstract is not necessary

Reply:
Good catch -- I didn't realize `abstract` was valid syntax in an interface.